tests/migration/test_migration.py

```diff
@@ -13,7 +13,8 @@
 @pytest.fixture
 def run_migrations(db_execute):
     def _(dbname):
-        migrations = sorted(pathlib.Path("procrastinate/sql/migrations").glob("*.sql"))
+        folder = pathlib.Path(__file__).parents[2] / "procrastinate/sql/migrations"
```

Comment:
So just to have it explicitely said, this allows the tests to be launched from a subfolder.